doc/tutorials/application-integration.md

```diff
@@ -185,12 +180,12 @@ Thus we are interested in the value we use:
 char *val = keyString(k);
 ```
 
-Finally, we need to convert the configuration value to the datatype we
+Now we need to convert the configuration value to the data-type we
```

Comment:
it doesn't display recommendations, it only complains that finally isn't really necessary. but you are right now isn't really necessary either imo its perfectly fine to start with We.
fixed data type.